Update UG (#242)
AY2122S1-CS2103T-T09-2/tp: docs/UserGuide.md

* Name, email and module code are **compulsory**
* Lesson code, phone number, telegram handle and remarks are **optional**
* You can add multiple module codes, and can have multiple lesson codes for a single module code

> * You will not be able to add a person with the same email/phone number/telegram handle

Reviewer: ```suggestion
* You will not be able to add a person with the same email/phone number/telegram handle as another existing contact in contHACKS
```

* At least one of the parameters must be provided
* Existing values of the parameters specified will be erased and updated to the input values
* You can input multiple module codes, and can have multiple lesson codes for a single module code

> * You will not be able to edit a person into a given email/phone number/telegram handle if it already exists in contHACKS

Reviewer: ```suggestion
* Edited person should not contain the same email/phone number/telegram handle as another existing contact in contHACKS
```

Author: for this i prefer my current version

* All the parameters are **compulsory** except for remark
* Day input is only accepted in integer form <br> e.g. `1` for Monday, `7` for Sunday
* Start and end time input is only accepted in the `HH:mm` format <br> e.g. `09:00`

> * You will not be able to add a lesson with the same module and lesson code

Reviewer: ```suggestion
* You will not be able to add a lesson with the same module and lesson code as another existing lesson in contHACKS
```

* At least one of the parameters must be provided.
* Existing values of the parameters specified will be erased and updated to the input values

> * You will not be able to edit a lesson into a given module and lesson code if it already exists in contHACKS

Reviewer: ```suggestion
* Edited lesson should not contain the same module and lesson code as another existing lesson in contHACKS
```
